mw-webapp/src/component/table/columns.tsx

```diff
@@ -99,13 +98,14 @@ export const columns: ColumnDef<DayReport, Date & JobDone[] & PlanForNextPeriod[
       );
     },
   }),
-  columnHelper.accessor<"studentComments", string[]>("studentComments", {
+  columnHelper.accessor<"studentComments", StudentComment[]>("studentComments", {
     header: "Student comments",
     cell: ({row}) => {
       const parentID = row.original.uuid;
+
       return (
         row.original.studentComments
-          ?.map((studentCommentItem) => (getStringArrayItem(studentCommentItem, parentID)))
+          ?.map((studentCommentItem) => (getStringArrayItem(studentCommentItem.description, parentID, studentCommentItem.isDone)))
       );
```

Comment:
If you changed model for student comments then you shouldn't use function getStringArrayItem. There exists function getObjectArrayItem for cells  that have custom type

Reply:
1. StudentComments don't have required uuid
2. I would need to change whole function to correctly display it

Reply:
Right? student comment hasn't uuid because it was just string[] not custom model

Reply:
Yes and you told me student comment will not need uuid. So should i completely change a function because of one model?

Reply:
So I think that studentComment no need uuid because I think that if studentComments is a string[] it's enough

But you changed string[] to custom model.

Reply:
So I think that studentComment no need uuid because I think that if studentComments is a string[] it's enough

But you changed string[] to custom model.

Reply:
This table component is really bad, also it is not a component (read docs), don't hesitate to change anything if it could improve coffee